pkg/datagatherer/k8s/generic.go

```diff
@@ -110,9 +112,37 @@ func (g *DataGatherer) Fetch() (interface{}, error) {
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
+	// Redact Secret data
+	err = redactList(list)
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
 	return list, nil
 }
 
+func redactList(list *unstructured.UnstructuredList) error {
+	// In principal we could only redact the list if it's kind is SecretList or
+	// a generic mixed List, however the test suite does not set the list kind
+	// and it is safer to always check for Secrets.
+	// Iterate over the items in the list.
+	for i := range list.Items {
+		// Determine the kind of items in case this is a generic 'mixed' list.
+		gvks, _, err := scheme.Scheme.ObjectKinds(&list.Items[i])
+		if err != nil {
+			return errors.WithStack(err)
+		}
+		for _, gvk := range gvks {
+			// If this item is a Secret then we need to redact it.
+			if gvk.Kind == "Secret" && (gvk.Group == "Core" || gvk.Group == "") {
```

Comment:
`Core` should be `core`

Reply:
(groups are always lowercase)

Reply:
😬 Fixing now!

Reply:
https://github.com/jetstack/preflight/pull/136